metabrainz/templates/index/about.html

```diff
@@ -1,66 +1,67 @@
+    <li>{{ _('Director Nick Ashton-Hart, Executive Director of the <a href="http://www.internet-ecosystem.org/people">Internet &amp; Digital Ecosystem Alliance</a>') }}</li>
+    <li>{{ _('Director <a href="http://paulbennun.com/">Paul Bennun</a> of <a href="http://www.somethinelse.com/">Somethin\' Else</a>') }}</li>
+    <li>{{ _('Director <a href="http://craphound.com/">Cory Doctorow</a>, Sci-Fi author and editor of the <a href="https://boingboing.net/">Boing Boing</a> weblog') }}</li>
+    <li>{{ _('Director <a href="https://www.linkedin.com/in/sophie-goossens-9094759">Sophie Goossens</a> of <a href="http://www.august-debouzy.com/">August &amp; Debouzy</a>') }}</li>
+    <li>{{ _('Director <a href="https://www.linkedin.com/pub/matthew-hawn/8/b74/8a">Matthew Hawn</a> of <a href="https://www.audionetwork.com/">Audio Network</a>') }}</li>
+    <li>{{ _('Director &amp; Treasurer <a href="https://www.linkedin.com/in/rassamih">Rassami Hok Ljungberg</a> of the <a href="http://www.music4point5.com/">Music 4.5 seminar series</a> put on by <a href="http://2pears.com/">2 Pears</a>') }}</li>
+    <li>{{ _('Executive Director <a href="http://mayhem-chaos.net/">Robert Kaye</a> of the MetaBrainz Foundation') }}</li>
+    <li>{{ _('Secretary <a href="https://freso.dk/">Frederik “Freso” S. Olesen</a>, Community Manager of the MetaBrainz Foundation') }}</li>
   </ul>
   <p>
- Our directors emeritus include:
+ {{ _('Our directors emeritus include:') }}
   </p>
   <ul>
-    <li>Director Emeritus <a href="http://lessig.org/">Lawrence Lessig</a> of the <a href="https://cyber.law.harvard.edu/people/llessig">Berkman Center for Internet and Society</a> and <a href="https://creativecommons.org/">Creative Commons</a></li>
-    <li>Director Emeritus <a href="https://joi.ito.com/">Joichi Ito</a> of <a href="http://www.neoteny.com/">Neoteny Co. Ltd.</a></li>
+    <li>{{ _('Director Emeritus <a href="http://lessig.org/">Lawrence Lessig</a> of the <a href="https://cyber.law.harvard.edu/people/llessig">Berkman Center for Internet and Society</a> and <a href="https://creativecommons.org/">Creative Commons</a>') }}</li>
+    <li>{{ _('Director Emeritus <a href="https://joi.ito.com/">Joichi Ito</a> of <a href="http://www.neoteny.com/">Neoteny Co. Ltd.</a>') }}</li>
   </ul>
 
-  <h3>Officers</h3>
-  The officers of this company are:
+  <h3>{{ _('Officers') }}</h3>
+  {{ _('The officers of this company are:') }}
   <ul>
-    <li>President Robert Kaye</li>
-    <li>Treasurer <a href="https://www.linkedin.com/in/rassamih">Rassami Hok Ljungberg</a></li>
-    <li>Secretary <a href="https://freso.dk/">Frederik “Freso” S. Olesen</a></li>
+    <li>{{ _('President') }} Robert Kaye</li>
+    <li>{{ _('Treasurer') }} <a href="https://www.linkedin.com/in/rassamih">Rassami Hok Ljungberg</a></li>
+    <li>{{ _('Secretary') }} <a href="https://freso.dk/">Frederik “Freso” S. Olesen</a></li>
```

Comment:
In some places you include names, in some you don't. This needs to be consistent.